trymito.io/components/CalendarDay/CalendarDay.module.css

```diff
@@ -0,0 +1,30 @@
+.calendar_day_card {
+    background-color: var(--color-light-background);
+    padding: 1rem;
+    text-align: center;
+
+    max-width: 320px;
+    flex-basis: 1;
+}
+
+.calendar_day_card > h2 {
+    color: var(--color-text-secondary);
+    margin-top: 1.5rem;
+}
+
+.calendar_day_card > p {
+    margin-top: .5rem;
+}
+
+@media only screen and (min-width: 50em) {
```

Comment:
<img width="401" alt="Screen Shot 2022-05-04 at 4 16 58 PM" src="https://user-images.githubusercontent.com/21002431/166827315-347a8277-295e-4387-a79c-65cbe1cd8702.png">

It doesn't look like the border radius gets applied when things are small. Do we want it to be?

Reply:
Yeah, because when they wrap, the Before ones turn into 1 long calendar, which I think is nice. But if they have rounded corners, then it looks bad.